src/actions/pubSubHandlers.js

```diff
@@ -244,7 +247,6 @@ export function handleRemoveRoom({ channel: id }) {
       payload: id,
     })
     if (id === currentId) dispatch(goTo('/chat'))
-    dispatch(handleCurrentUserLeftChannel())
```

Comment:
This was miss-leading, because even when the user is kicked from a channel the event `channel.left` comes in and not `channel.removed`. This means we always only need to go to `/chat` if it's the current channel.